src/main/java/cashleh/commands/Command.java

```diff
@@ -10,7 +10,8 @@
  */
 public class Command {
     protected Logger logger = Logger.getLogger("CommandLogger");
-    protected Level loggerLevel = Level.OFF;
+    protected Level loggerLevel = Level.INFO;
+    protected boolean islogging = false;
```

Comment:
Nice way of handling this. Maybe in the next version we could create a separate file containing all the logging information